go.mod

```diff
@@ -4,7 +4,7 @@ go 1.15
 
 require (
 	github.com/BurntSushi/toml v0.3.1
-	github.com/containers/image/v5 v5.13.2
+	github.com/containers/image/v5 v5.13.3-0.20210716170444-d695b98f8381
```

Comment:
@mtrmac can we release a new c/image version to make this bump sane?

Reply:
I’d much prefer to fix https://github.com/containers/common/pull/659#issuecomment-875505807 first, so that there isn’t a c/image release with the extremely surprising “log into `a.com/b` provides credentials for `a.com/c`” behavior.

Reply:
Sounds good, working on it in https://github.com/containers/image/pull/1304